frontend/apps/reader/modules/readermenu.lua

```diff
@@ -81,13 +81,13 @@ function ReaderMenu:onReaderReady()
     self.ui:registerTouchZones({
         {
             id = "readermenu_tap",
-            ges = "tap",
+            ges = "swipe",
             screen_zone = {
                 ratio_x = DTAP_ZONE_MENU.x, ratio_y = DTAP_ZONE_MENU.y,
                 ratio_w = DTAP_ZONE_MENU.w, ratio_h = DTAP_ZONE_MENU.h,
             },
             overrides = { "tap_forward", "tap_backward", },
```

Comment:
These overrides are not accurate anymore. Need to update to `rolling_swipe` and `paging_swipe`.